models/stg_iterable__user_device_history.sql

```diff
@@ -0,0 +1,44 @@
+{{ config(enabled=var('iterable__using_user_device_history', True)) }}
```

Comment:
I thought the default for this was supposed to be `False`?

Reply:
ah yes will change